numpy/core/src/umath/loops_trig.dispatch.c.src

```diff
@@ -0,0 +1,235 @@
+/*@targets
+ ** $maxopt $werror baseline
+ ** (avx2 fma3) avx512f
+ ** vsx2
+ ** neon_vfpv4
+ **/
+#include "numpy/npy_math.h"
+#include "simd/simd.h"
+#include "loops_utils.h"
+#include "loops.h"
+/*
+ * TODO:
+ * - use vectorized version of Payne-Hanek style reduction for large elements or
+ *   when there's no native FUSED support instead of fallback to libc
+ */
+#if NPY_SIMD_FMA3 // native support
+/**
+ * IDK, what's wrong on MSVC, it seems large SIMD kernels causes ambiguous link errors
+ * which lead to break the build, probably releated to the following bug:
+ * https://developercommunity.visualstudio.com/content/problem/415095/internal-compiler-error-with-perfectly-forwarded-r.html
+ */
+#ifdef _MSC_VER
+    #define SIMD_MSVC_NOINLINE __declspec(noinline)
```

Comment:
check if msvc2019 can pass the build without noinline

Reply:
I tested it on msvc2019 and it seemed to work fine also in the newest build of msvc2017, I moved this #def to loops_utils.h.src